wallet/src/actors/app/handlers/create_data_req.rs

```diff
@@ -11,6 +11,7 @@ pub struct CreateDataReqRequest {
     wallet_id: String,
     request: DataRequestOutput,
     fee: u64,
+    fee_type: types::FeeType,
```

Comment:
```suggestion
    fee_type: Option<types::FeeType>,
```
By default it could be weighted fee, so that we don't break the API :)